# PlaySong.srv
uint8 song_number
---
bool success
